docs/topics/native/native-gradle.md

```diff
@@ -3,117 +3,131 @@
+   plugins {
+       kotlin("multiplatform") version "%kotlinVersion%"
+   }
+
+   repositories {
+       mavenCentral()
+   }
+
+   kotlin {
+     macosX64("native") { // on macOS
+     // linuxX64("native") // on Linux
+     // mingwX64("native") // on Windows
+       binaries {
+         executable()
+       }
+     }
+   }
+
+   tasks.withType<Wrapper> {
+     gradleVersion = "%gradleVersion%"
+     distributionType = Wrapper.DistributionType.BIN
+   }
+   ```
 
 </tab>
 <tab title="Groovy" group-key="groovy">
 
-```groovy
-plugins {
-    id 'org.jetbrains.kotlin.multiplatform' version '%kotlinVersion%'
-}
-
-repositories {
-    mavenCentral()
-}
-
-kotlin {
-  macosX64('native') { // on macOS
-  // linuxX64('native') // on Linux
-  // mingwX64('native') // on Windows
-    binaries {
-      executable()
-    }
-  }
-}
-
-wrapper {
-  gradleVersion = '%gradleVersion%'
-  distributionType = 'BIN'
-}
-```
+   ```groovy
+   // build.gradle
+   plugins {
+       id 'org.jetbrains.kotlin.multiplatform' version '%kotlinVersion%'
+   }
+
+   repositories {
+       mavenCentral()
+   }
+
+   kotlin {
+     macosX64('native') { // on macOS
+     // linuxX64('native') // on Linux
+     // mingwX64('native') // on Windows
+       binaries {
+         executable()
+       }
+     }
+   }
+
+   wrapper {
+     gradleVersion = '%gradleVersion%'
+     distributionType = 'BIN'
+   }
+   ```
 
 </tab>
 </tabs>
 
-Next, create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+   You can use different [target presets](mpp-supported-platforms.md), such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
+   to define corresponding the target platform. The preset name describes a platform for which you are compiling your code.
+   These target presets optionally take the target name as a parameter, which is `native` in this case.
+   The target name is used to generate the source paths and task names in the project.
+
+3. Create an empty `settings.gradle` or `settings.gradle.kts` file in the project folder.
+
+4. Create a folder `src/nativeMain/kotlin` and place inside the `hello.kt` file with the following content:
+   
+   ```kotlin
+   fun main() {
+     println("Hello, Kotlin/Native!")
+   }
+   ```
 
-Depending on the target platform, different [functions](mpp-supported-platforms.md),
-such as `macosX64`, `mingwX64`, `linuxX64`, `iosX64`,
-are used for creating the Kotlin target. The function name is the platform for which you are compiling your code. 
-These functions optionally take the target name as a parameter, which is `"native"` in our case. 
-The specified _target name_ is used to generate the source paths and task names in the project.  
+   By convention, all sources are located in the `src/<target name>[Main|Test]/kotlin` folders, where `main` is for the source code
+   and `test` is for tests. `<target name>` corresponds to the target platform (in this case `native`), as specified in the build file.
```

Comment:
Is it correct to call the given optional name ('native') a target platform?

Reply:
Maybe expand the explanation and add an example if how it looks when the name is not specified

Reply:
Decided to come back to it later